Projects/TerminateContracts/Sources/Views/ConfirmTerminationScreen.swift

```diff
@@ -0,0 +1,103 @@
+import SwiftUI
+import hCore
+import hCoreUI
+
+struct ConfirmTerminationScreen: View {
+    @PresentableStore var store: TerminationContractStore
+    let config: TerminationConfirmConfig?
```

Comment:
Don't do this as optional. We want to display this screen only if we have config